src/liquidity.rs

```diff
@@ -892,3 +895,113 @@ pub(crate) struct LSPS2BuyResponse {
 	intercept_scid: u64,
 	cltv_expiry_delta: u32,
 }
+
+/// A liquidity handler allowing to request channels via the [bLIP-51 / LSPS1] protocol.
+///
+/// Should be retrieved by calling [`Node::lsps1_liquidity`].
+///
+/// [bLIP-51 / LSPS1]: https://github.com/lightning/blips/blob/master/blip-0051.md
+/// [`Node::lsps1_liquidity`]: crate::Node::lsps1_liquidity
+#[derive(Clone)]
+pub struct LSPS1Liquidity {
```

Comment:
Given LSPS2 channels are JIT, I suppose a similar struct won't be needed there? Perhaps the docs should mention how JIT channels are opened.

Reply:
Yeah, though perhaps a single liquidity handler would be sufficient (i.e., no need for users to care which BLIP the method is defined in). I'm indifferent as to whether we should keep the method in the `Bolt11Payment` handler. But we should probably cross-link the docs where appropriate.

Reply:
Right, I figure there at least be one abstraction for the grouping for LSP spec and method naming can differentiate as needed. But if there are competing standards inside that (or the move to BLIPS obsoletes the grouping), then maybe a separate one is needed.